x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group3/builtin_alert_types/es_query/esql_only.ts

```diff
@@ -260,6 +250,7 @@ export default function ruleTests({ getService }: FtrProviderContext) {
       expect(alertDoc['host.name']).to.eql(['host-1']);
       expect(alertDoc['host.hostname']).to.eql(['host-1']);
       expect(alertDoc['host.id']).to.eql(['1']);
+      expect(alertDoc.c[0]).greaterThan(0);
```

Comment:
Does this mean a query can overwrite ANY field in the alertDoc?  And I guess add new fields not defined in mappings?  Neither of those seem right ...

Reply:
I wanted to acknowledge for the PR that we are talking offline about this 🙂